### PR TITLE
Improve seeding with realistic users, channels, visit counts and activity.

### PR DESCRIPTION
**Pull request by jonsugar**
This PR changes the seeders to create a more realistic feeling forum. 

- Four users are created.
- Realistic channels are created (Eloquent, Laravel Mix, PHP, VueEx and VueJS )
- The seeded threads are assigned to the existing channels and users (currently new users and channels are created for each thread).
- Each thread is seeded with a visit count
- A seeded thread may be locked (there is a 15% chance a thread will be created locked)
- Thread creation activity is recorded so that user profiles are correct (currently seeded users have no activity)

screenshot-2018-2-15 council

### CODE REVIEW

File: database/seeds/SampleDataSeeder.php [hunk 1]
```diff
@@ -32,7 +32,45 @@ protected function channels()
...
+                'archived' => false,
+                'color' => '#a01212'
+            ],
+            [
+                'name' => 'VueEx',
+                'description' => 'This channel is for all VueEx specific questions.',
+                'archived' => false,
+                'color' => '#ff8822'
+            ],
+        ])->each(function ($channel) {
```
Comment (ghazanfarmir) [inline database/seeds/SampleDataSeeder.php]: each() is deprecated since 7.2

http://php.net/manual/en/function.each.php

Comment (ruman63) [reply database/seeds/SampleDataSeeder.php]: This each method is from laravel's Collection.

**Comment by MichaelDeBoey** [comment]
I think it's best to rebase onto `master` 🙂

**Comment by jonsugar** [comment]
Hi, @MichaelDeBoey are you talking about removing all the `wip` commits? If so, they are annoying me also.

I'm not a git master just yet 😁. Can you just point me in the right direction and I will do it.

**Comment by MichaelDeBoey** [comment]
@jonsugar Yeah that's what I'm talking about indeed :-)

First set `JeffreyWay/council` as the `upstream`
```sh
git remote -v
git remote add upstream https://github.com/JeffreyWay/council
git remote -v
```

And fetch it down
```sh
git fetch upstream
```

Make sure master is the same as `upstream`
```sh
git checkout master
git merge upstream/master
git push
```

Rebase `better-seeding` onto `master`
```sh
git checkout better-seeding
git rebase master
```

Now check if only your commits are on the `better-seeding` branch.

If so, `force push` it to `origin`
```sh
git push --force
```

**Pull request merged**